examples/source-database/mysql-deployment.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mysql
+  labels:
+spec:
+  selector:
+    matchLabels:
+      tier: mysql
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        tier: mysql
+    spec:
+      containers:
+        - image: quay.io/rcook/tools:mysql
```

Comment:
Should we use an official mysql container?

Reply:
I had to rehost it due to running on a RH network that hit the dockerhub limit. Will change